src/api/reflector.rs

```diff
@@ -132,18 +129,15 @@ where
 
     /// Read data for users of the reflector
     ///
-    /// TODO: deprecate in favour of a stream returning fn..
+    /// TODO: deprecate in favour of a stream returning fn, this would also get rid of the
+    /// intermediate future
```

Comment:
Will kill this comment :-)
Stream returning function isn't really a thing we need in Reflector. That's the Informer's job.